README.md

```diff
@@ -49,102 +50,144 @@ See below for build and launch instructions.
+
+This should be enough to start the game and reach the main menu and from there, join a server. If the server supports autodownload mechanism, Unvanquished will fetch all the missing packages from it.
+
+If you are looking for the sources of the whole assets, have a look at the [UnvanquishedAssets](https://github.com/UnvanquishedAssets/UnvanquishedAssets) repository. Beware, unlike the `unvanquished_src.dpkdir` package, most of them can't be loaded correctly by the engine without being built first.
+
+#### Loading your own assets
 
 As a developer, you will want to load your own assets in addition to those shipped with the game. To do that:
 
-    cd build
-    mkdir -p pkg; cd pkg
-    ln -s ../../main git_source.pk3dir
-    mkdir assets_source.pk3dir
-    echo git > assets_source.pk3dir/DEPS
+```sh
+cd build
+mkdir -p pkg && cd pkg
+mkdir assets_src.dpkdir
+```
+
+You can now put loose assets into `assets_src.dpkdir` or you can put additional dpkdir directories or dpk containers inside `pkg` and add their names (the format is `<NAME>_<VERSION>.dpk[dir]`) as lines to the `DEPS` file (the format is `<NAME> <VERSION>`). Version is required in package filename but optional in `DEPS` file. In order to launch Unvanquished, use one of the following commands:
+
+```sh
+# Runs the game and loads the “assets” package and its dependencies from “pkg/” directory:
+./daemon -set fs_extrapaks assets
 
-You can now put loose assets into `assets_source.pk3dir` or you can put additional pk3dir directories or pk3 containers inside `pkg` and add their names (the format is `NAME_VERSION.pk3(dir)`) as lines to the `DEPS` file. In order to launch Unvanquished, use one of the following commands:
+# Runs the game and loads the “assets” package and its dependencies from <PATH>
+# when <PATH> is not one of the default Unvanquished paths:
+./daemon -pakpath <PATH> -set fs_extrapaks assets
 
-  - `./daemon -pakpath PATH -set fs_extrapaks assets # ¹`
-  - `./daemon -pakpath PATH -set fs_extrapaks assets -set vm.sgame.type 3 -set vm.cgame.type 3 -set vm.sgame.debug 1 -set vm.cgame.debug 1 +devmap plat23 # ²`
+# In addition, load a shared-object gamelogic you compiled and allow it to be debugged,
+# then launch the map “Platform 23” with cheats enabled after startup:
+./daemon -pakpath <PATH> -set fs_extrapaks assets \
+			-set vm.sgame.type 3 -set vm.cgame.type 3 \
+			+devmap plat23
 
-¹ *Runs the game and loads the `assets` package and its dependencies. `PATH` is the path to Unvanquished's base packages and maps. Omit `-pakpath PATH` if `pkg` contains these assets.*
-² *In addition, load a shared-object gamelogic you compiled and allow it to be debugged. Launch the map Platform 23 with cheats enabled after startup.*
+# Same but load an NaCl gamelogic you compiled and allow it to be debugged:
```

Comment:
Using the DLL is the best option for a developer 95% of the time (besides the easier debugging, it also builds much faster), so I think rarely needed info about nacl debugging should be relegated to the wiki instead of appearing on the main page of the readme.

Reply:
sounds fine

Reply:
I removed thes lines to make the readme lighter